parts/k8s/cloud-init/artifacts/mountetcd.sh

```diff
@@ -12,7 +12,7 @@ if mount | grep $MOUNTPOINT; then
   umount /dev/sdc1
 fi
 if ! grep "/dev/sdc1" /etc/fstab; then
-  echo "$PARTITION       $MOUNTPOINT       auto    defaults,nofail       0       2" >>/etc/fstab
+  echo "LABEL=etcd_disk       $MOUNTPOINT       auto    defaults,nofail       0       2" >>/etc/fstab
```

Comment:
nit: we should make `etcd_disk` a var like `PARTITION` and `MOUNTPOINT`

Reply:
All of these changes are happening here: https://github.com/Azure/aks-engine/pull/3259

The purpose of this PR is to change as few things as possible to deliver the fix for reboots getting different OS devices mapped to underlying disks

Reply:
I'm fine with that, moving to a var is just better for code safety so we don't accidentally change one reference to it but not all. My other comment is required for this to work though, with the change as-is a new line will be added every time you reboot.